doc/sphinx/source/project/introduction.rst

```diff
@@ -166,5 +166,12 @@ and nGraph.
    Figure 7: A closer look at tensor compilers.
 
 
+Other notable efforts
+----------------------
+
+
+See: https://arxiv.org/pdf/1805.00907.pdf 
```

Comment:
Should also mention at least XLA and TVM, and we should probably provide detailed citations rather than a bare URL.

Reply:
I added a short list for now, we can always expand later.